fairlearn/reductions/_moments/regression_moment.py

```diff
@@ -0,0 +1,134 @@
+from .utility_parity import UtilityParity
+from .moment import Moment,_LABEL,_GROUP_ID
+from .weighted_error_rate import WeightedErrorRate
+from .bounded_group_loss import SquareLoss
+import pandas as pd
+import numpy as np
+from itertools import repeat
+
+class CDF_DemographicParity(Moment):
```

Comment:
Should this inherit from LossMoment?

Reply:
I chose not to make it inherited from LossMoment because the comments about LossMoment say that "Moment that can be expressed as weighted loss". It seems that CDF_DemographicParity does not fit into this.